### PR TITLE
Add topic interface

### PR DESCRIPTION
**Pull request by yuokamoto**
depends this pr
https://github.com/rapyuta-robotics/kamajii_common/pull/51

### CODE REVIEW

File: grippers/rr_suction_pump/include/rr_suction_pump/suction_pump.hpp [hunk 1]
```diff
@@ -69,6 +59,37 @@ class Pump
...
+class PumpWithFeedback : public Pump<HI, Config>
+{
+public:
+    typedef std::unique_ptr<HI> HI_ptr;
+    PumpWithFeedback(const std::vector<std::string> triggerPins, const int numOfTrigger, const std::vector<std::string> statusPins, const int numOfStatus,
+            const bool outputNormallyOn, const bool inputNormallyOn)
+            : Pump<HI, Config>(triggerPins, numOfTrigger, outputNormallyOn), _inputNormallyOn(inputNormallyOn)
+    {
+        for (int i = 0; i < numOfStatus; i++) {
+            _status.push_back(HI_ptr(new HI(statusPins[i], HI::Type::RR_HW_INTERFACE_INPUT)));
```
Comment (Dale-Koenig) [inline grippers/rr_suction_pump/include/rr_suction_pump/suction_pump.hpp]:
```suggestion
            _status.emplace_back(new HI(statusPins[i], HI::Type::RR_HW_INTERFACE_INPUT));
```
saves a copy and text if you care (similar with other places)

File: grippers/rr_suction_pump/src/suction_pump_topic_driver.cpp [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            for (int j = 0; j < trigger_num; j++) {
+                nh.getParam("suction_pump" + std::to_string(i + 1) + "/trigger_gpio" + std::to_string(j), trigger);
+                trigger_gpio.push_back(trigger);
+                ROS_INFO_STREAM_NAMED(_topic_name, "trigger: " << trigger);
+            }
+            _pumps.push_back(std::unique_ptr<Pump<HI, Config>>(
+                    new Pump<HI, Config>(trigger_gpio, trigger_num, outputNormallyOn)));
+        }
+        
+        _suction_pump_cmd_sub = nh.subscribe(topic_name, 10, &SuctionPumpTopicDriver<HI, Config>::cb, this);
```
Comment (Dale-Koenig) [inline grippers/rr_suction_pump/src/suction_pump_topic_driver.cpp]:
```suggestion
        _suction_pump_cmd_sub = nh.subscribe(topic_name, 1, &SuctionPumpTopicDriver<HI, Config>::cb, this);
```
We always want the most recent message right?

Comment (yuokamoto) [reply grippers/rr_suction_pump/src/suction_pump_topic_driver.cpp]: yes

Comment (Dale-Koenig) [reply grippers/rr_suction_pump/src/suction_pump_topic_driver.cpp]: then we probably want to not allow a queue

**Pull request merged**